LEInstaller/Form1.cs

```diff
@@ -58,6 +58,7 @@ private void Form1_Load(object sender, EventArgs e)
 
             buttonUninstall.Enabled = ShellExtensionManager.IsInstalled(false);
             buttonUninstallAllUsers.Enabled = ShellExtensionManager.IsInstalled(true);
+            buttonEditGlobal.Enabled = ShellExtensionManager.IsInstalled(true);
```

Comment:
`IsInstalled(true)` only cares about the installation for all users in system. You should use `IsInstalled(true) || IsInstalled(false)`.